sale_input_barcode/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_
```

Comment:
a reason to set an underscore at the end of the file ?

Reply:
test_.py contains test for this module. This is the minimal name required else test unit is not triggered.
The module is simple, no need several files. In this case there is no need to have a longer name.
Logging is self explained

![Screenshot from 2022-03-14 09-29-06](https://user-images.githubusercontent.com/1853434/158133821-a77d1d59-82d3-47b9-b29d-1fcaf0c2200a.png)

At least it works as it

Reply:
it doesn't work if the name is ``test.py`` ?

Reply:
On some Odoo versions at least. I may test again with `test.py` if you prefer ! ? while it's not useless long name, no problem

Reply:
I checked once again it's not triggered if only called `test.py`

Reply:
thanks. I didn't know.